src/PHPUnit/NonFinal/JUnit.php

```diff
@@ -324,11 +325,9 @@ public function endTest(Test $test, float $time): void
     {
         $numAssertions = 0;
 
-        if (\method_exists($test, 'numberOfAssertionsPerformed')) {
-            // PHPUnit 10
+        if (PHPUnit10::numberOfAssertionsPerformedMethodExists($test)) {
             $numAssertions = $test->numberOfAssertionsPerformed();
         } else {
-            // PHPUnit 9
             $numAssertions = $test->getNumAssertions();
```

Comment:
`getNumAssertions` method does not exist in PHPUnit 9 only.
it is defined in [AssertionCounter trait](https://github.com/Codeception/Codeception/blob/5.0/src/Codeception/Test/Feature/AssertionCounter.php) too.

Reply:
So the whole conditional isn't necessary?

Reply:
this is  `// PHPUnit 9 or Cest or Cept or Gherkin`

Reply:
Done.